pkg/registry/service/portallocator/controller/repair.go

```diff
@@ -86,10 +86,8 @@ func (c *Repair) runOnce() error {
 	if err != nil {
 		return fmt.Errorf("unable to refresh the port block: %v", err)
 	}
-
 	ctx := api.WithNamespace(api.NewDefaultContext(), api.NamespaceAll)
-	options := &api.ListOptions{ResourceVersion: latest.ObjectMeta.ResourceVersion}
-	list, err := c.registry.ListServices(ctx, options)
+	list, err := c.registry.ListServices(ctx, nil)
```

Comment:
OK - got it; you're right of course.

@lavalamp - do you want to merge this PR or do it exactly as you did in:
https://github.com/lavalamp/kubernetes/commit/5d713bb6aab4a3cf75e03f4cfa67b8398db34004
(i.e. add extended comment about it)